src/jobs/models/jobManager.js

```diff
@@ -276,18 +279,23 @@ async function createJobRequest(jobId, reportId, jobBody, dockerImage, configDat
 
 function addCron(jobId, job, cronExpression, configData) {
     const scheduledJob = new CronJob(cronExpression, async function () {
+        let report = null;
         try {
             if (job.enabled === false) {
                 logger.info(`Skipping job with id: ${jobId} as it's currently disabled`);
             } else {
                 const latestDockerImage = await dockerHubConnector.getMostRecentRunnerTag();
                 const test = await testsManager.getTest(job.test_id);
-                const report = await createReportForJob(test, job);
+                report = await createReportForJob(test, job);
```

Comment:
Is it possible to extract these functions to be common?

Reply:
I do agree we need to extract the logic to a function